### PR TITLE
Prevent exception in events.js when data is null

### PR DESCRIPTION
**Pull request by peterkinmond**
*Issue #, if available:* 

https://github.com/amazon-connect/amazon-connect-streams/issues/372

*Description of changes:*

After upgrading to v1.6.3 we started seeing this error come up in the Chrome dev console which was due to a recent addition to the `EventBus.prototype.trigger` method. 

Screen Shot 2021-02-03 at 4 25 43 PM

This PR adds a null check which should prevent that error.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/event.js [hunk 1]
```diff
@@ -286,7 +286,7 @@
       connect.getLog().trace("Publishing event: %s", eventName).sendInternalLogToServer();
     }
 
-    if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data.contactId && !(data instanceof connect.Contact)) {
+    if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data !== null && data.contactId && !(data instanceof connect.Contact)) {
```
Comment (ctwomblyamzn) [inline src/event.js:289]: Any reason that this isn't just `if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data && data.contactId && !(data instanceof connect.Contact)) {`

Comment (peterkinmond) [reply src/event.js:289]: Nope! I tried to keep it consistent with existing code and there were several instances of using `!== null` like [this](https://github.com/amazon-connect/amazon-connect-streams/blob/master/src/aws-client.js#L5179) and [this](https://github.com/amazon-connect/amazon-connect-streams/blob/master/src/aws-client.js#L6796). I'm happy to change it though 😀
WDYT?

Comment (ctwomblyamzn) [reply src/event.js:289]: A fair point, I think `&& data &&` is cleaner. But I appreciate that you are trying to keep the codebase consistent, and it doesn't do any harm, so I'm going to approve.

**Review by ctwomblyamzn — APPROVED** [review]

**Review by h3adache — APPROVED** [review]

**Comment by peterkinmond** [comment]
Great thanks @ctwomblyamzn! Do you know when the next release will be? We're currently blocked on upgrading to latest from the bug that this PR fixes

**Comment by ctwomblyamzn** [comment]
It should be this week @peterkinmond

**Pull request merged**